pkger/service.go

```diff
@@ -333,6 +335,184 @@ func (s *Service) DeleteStack(ctx context.Context, identifiers struct{ OrgID, Us
 	return s.store.DeleteStack(ctx, identifiers.StackID)
 }
 
+func (s *Service) ExportStack(ctx context.Context, orgID, stackID influxdb.ID) (*Pkg, error) {
+	stack, err := s.store.ReadStackByID(ctx, stackID)
+	if err != nil {
+		return nil, err
+	}
+
+	labelObjs, availablePkgLabels, err := s.exportStackLabels(ctx, stack.Resources)
+	if err != nil {
+		return nil, err
+	}
+
+	pkg := Pkg{Objects: labelObjs}
+	for _, res := range stack.Resources {
+		var (
+			obj Object
+			err error
+		)
+
+		switch res.Kind {
+		case KindBucket:
+			bkt, err := s.bucketSVC.FindBucketByID(ctx, res.ID)
+			if influxdb.ErrorCode(err) == influxdb.ENotFound {
+				continue
+			}
+			if err != nil {
+				return nil, ierrors.Wrap(err, fmt.Sprintf("failed to find bucket[%s]", res.ID.String()))
```

Comment:
so this failure message will say:

>failed to find bucket[12345876]

or is [%s] some special notation?

Reply:
you nailed it. The error message isn't great. I want to go through this service and update the error messages as part of our templates work. For some reason that use case feels like a good time to fixup error messaging... they platform errors we return could dramatically improve across teh platform, we can start with pkger here to try and touch that up 👍